providers/base/units/stress/s3s4.pxu

```diff
@@ -150,3 +150,221 @@ requires: cpuinfo.platform in ("i386", "x86_64")
 command: [ -e "$PLAINBOX_SESSION_SHARE"/hibernate_{s4_iterations}_cycles.log ] && cat "$PLAINBOX_SESSION_SHARE"/hibernate_{s4_iterations}_cycles.log
 _description:
  Attaches the log from the {s4_iterations} cycles Hibernate/Resume test if it exists
+
+id: stress_s3_cycles_iterations_1
```

Comment:
What's the rationale behind this id?
After reading the summary it seems that this is used to generate concrete iteration jobs.
Why not call it as such, for instance. `stress_s3_cycles_iteration_generator`

Reply:
Based on the structure in the [card](https://warthogs.atlassian.net/browse/CQT-2478) as below:
![Screenshot from 2023-08-10 17-56-07](https://github.com/canonical/checkbox/assets/47034756/18e6eb11-562e-4ae4-ac91-a102def6d7c3)

It need to identify the ` suspend_cycles_1_reboot{2..k}` and `suspend_cycles_{2…n}_reboot{1..k}` , and using two different resource `stress_s3_cycles_iterations_1 ` and ` stress_s3_cycles_iterations_multiple`.
I use the the suffix "_1" and "_multiple" to identify them, and if you have any suggestion for suffix of these two resource job.